pkg/cmd/deploy/manifest.go

```diff
@@ -154,7 +154,7 @@ func (manifest *Manifest) Interpreted(f *cmdutil.Factory, cmd *DeployCmd, conf *
 		if !conf.RulesEngine.Created {
 			// create rules engines to compute else delivery
 			if strings.ToLower(conf.Mode) == "compute" {
-				requestRules, err := requestRulesEngineManifest(conf.Origin.StorageOriginID, conf.Function.InstanceID, route)
+				requestRules, err := requestRulesEngineManifest(conf, conf.Function.InstanceID, route)
```

Comment:
if you're sending the conf structure, you do not need to send conf.Function.InstanceID separately